<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>589</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
background-color:rgb(9, 21, 36);
width: 320px;
heigth :500px
}
font {
color: white
}
QMenuBar, QMenu, QAction {
background-color: rgb(34, 41, 48);
color: rgb(101, 116, 133);
}

QAction:hover {
	color: white;
}
QMenuBar.QMenu.QAction:pressed {
	background-color: black;
}

QLabel {
font-size: 16px;
color: #fff;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>261</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>511</width>
      <height>101</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>420</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>533</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuColor_mode">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuColor_mode_2">
     <property name="title">
      <string>Color mode</string>
     </property>
     <addaction name="actionNegative"/>
     <addaction name="actionSepia"/>
     <addaction name="actionWhite_black"/>
     <addaction name="actionBlack_white"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="menuColor_mode_2"/>
    <addaction name="separator"/>
    <addaction name="actionNorm"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuDeep_Learning">
    <property name="title">
     <string>DL</string>
    </property>
    <widget class="QMenu" name="menuExample">
     <property name="title">
      <string>Example</string>
     </property>
     <addaction name="actionFind_Face"/>
     <addaction name="actionDetect_text_2"/>
    </widget>
    <addaction name="actionFind_Fac"/>
    <addaction name="actionDetect_text"/>
    <addaction name="separator"/>
    <addaction name="menuExample"/>
   </widget>
   <widget class="QMenu" name="menuHelp_me">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_me"/>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuColor_mode"/>
   <addaction name="menuDeep_Learning"/>
   <addaction name="menuHelp_me"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionSepia">
   <property name="text">
    <string>Sepia</string>
   </property>
  </action>
  <action name="actionNegative">
   <property name="text">
    <string>Negative</string>
   </property>
  </action>
  <action name="actionBlack_white">
   <property name="text">
    <string>Black-white</string>
   </property>
  </action>
  <action name="actionGray">
   <property name="text">
    <string>Gray</string>
   </property>
  </action>
  <action name="actionFind_Fac">
   <property name="text">
    <string>Find Face</string>
   </property>
  </action>
  <action name="actionDetect_text">
   <property name="text">
    <string>Detect text</string>
   </property>
  </action>
  <action name="actionFind_Face">
   <property name="text">
    <string>Find Face</string>
   </property>
  </action>
  <action name="actionDetect_text_2">
   <property name="text">
    <string>Detect text</string>
   </property>
  </action>
  <action name="actionNorm">
   <property name="text">
    <string>Normal</string>
   </property>
  </action>
  <action name="actionWhite_black">
   <property name="text">
    <string>White-black</string>
   </property>
  </action>
  <action name="actionAbout_me">
   <property name="text">
    <string>About me</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Directory</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Asks</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>edit mode</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>481</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>486</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
